media-plugins/mythplugins/mythplugins-30.0.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="xml"
+BACKPORTS="03f44039848bd09444ff4baa8dc158bd61454079"
+MY_P=${P%_p*}
+
+inherit eutils python-single-r1 readme.gentoo-r1 vcs-snapshot
```

Comment:
I don't see eutils being used here, and I doubt you need vcs-snapshot either (granted, didn't test this yet).

Reply:
I removed them and everything builds